src/views/ApartmentApp/components/StudentApplication/components/Agreements/index.js

```diff
@@ -19,51 +20,63 @@ import {
  * @returns {JSX.Element} JSX Element for the instructions card
  */
 const Agreements = ({ onChange }) => {
-  const [checkboxes, setCheckboxes] = React.useState([
-    {
-      checked: false,
-      label: 'Each individual on the application has agreed to be on the application',
-    },
-    {
-      checked: false,
-      label:
-        'We understand that if someone on this application has not agreed to be on the application, our application will be disqualified',
-    },
-    {
-      checked: false,
-      label: 'Each individual on this application appears ONLY on this application',
-    },
-    {
-      checked: false,
-      label:
-        "We understand that if an individual on this application also appears on another group's application, our application could be disqualified",
-    },
-    {
-      checked: false,
-      label:
-        'Any individual on this application who has been on disciplinary probation at any point during the 2019-2020 academic year has been approved to apply by the Dean of Student Care or the Director of Residence Life',
-    },
-    {
-      checked: false,
-      label:
-        'Each individual on this application intends to register as a full-time student by apartment selection night (Apr. 23)',
-    },
-    {
-      checked: false,
-      label:
-        'We understand that if any member of our application fails to register as a full-time student by Apr. 23, our application could be disqualified',
-    },
-    {
-      checked: false,
-      label:
-        'We have read and understand all of the information and guidelines listed in the Instructions section',
-    },
-    {
-      checked: false,
-      label:
-        'We certify that all information provided on this application is accurate, to the best of our knowledge',
-    },
-  ]);
+  const [checkboxes, setCheckboxes] = useState([]);
+
+  useEffect(() => {
+    const loadAgreements = async () => {
+      const currentYear = Number(new Date().getFullYear);
```

Comment:
I don't think you need the explicit conversion to a number here, but you do definitely need to call the `getFullYear` method rather than simply referencing it:
```suggestion
      const currentYear = new Date().getFullYear();
```

Reply:
I think I added that conversion because I didn't know why it VSCode was detecting it as the wrong thing. Now I know that it must have been because I forgot the `()` after the method call